pages/AddSurveyQuestionCommonUtils.js

```diff
@@ -0,0 +1,58 @@
+const { expect } = require("@playwright/test");
+
+export async function reset(page){
```

Comment:
```suggestion
export async function resetQuestion(page){
```
The function name should be descriptive and short at the same time.